app/controllers/rfid_controller.rb

```diff
@@ -14,6 +14,7 @@ def card_number
         render json: { success: "RFID exist" }, status: :ok
         check_session(rfid)
       else
+        rfid.update(mac_address: params[:mac_address])
```

Comment:
Why do `rfid.update` instead of just `rfid.mac_address = params[:mac_address]; rfid.save`? You can also drop the `touch` after this because updating the record will update the update_at which is what touch does

Reply:
i need `.touch` because if the mac_address is the same then `updated_at` won't update. 

>`test "posting existing card with no user updates updated_at" ` 
complains when i just do `.save` without `.touch` before it.